packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts

```diff
@@ -105,6 +110,48 @@ export class TelegramTrigger implements INodeType {
 				default: [],
 				description: 'The update types to listen to.',
 			},
+			{
+				displayName: 'Additional Fields',
+				name: 'additionalFields',
+				type: 'collection',
+				placeholder: 'Add Field',
+				default: {},
+				options: [
+					{
+						displayName: 'Download Images/Files',
+						name: 'donwload',
+						type: 'boolean',
+						default: false,
+						description: `Telegram develiers the image in 3 sizes.<br>
+						By default, just the larger image would be downloaded.<br>
+						if you want to change the size set the field 'Image Size'`,
+					},
+					{
+						displayName: 'Image Size',
+						name: 'imageSize',
```

Comment:
Should it not only be displayed if "download" is true?

Reply:
Yes, will add the fix tomorrow. Thanks